Dockerfile

```diff
@@ -37,7 +37,7 @@ RUN useradd -u 1000 -s /bin/bash mlserver -d /opt/mlserver && \
 
 COPY --from=wheel-builder /opt/mlserver/dist ./dist 
 RUN pip install --upgrade pip wheel setuptools && \
-    pip install ./dist/mlserver-*.whl[all]; \
+    pip install $(ls ./dist/mlserver-*.whl)[all]; \
```

Comment:
Should we add a note of why is the fix required? Mainly to ensure no one (probably us!) sees this in a couple months, thinks "what the heck?" and decides to remove it.

Reply:
done